### ISSUE TITLE
next/image image not loading : getting 503 (Service Unavailable) in simple image example

### ISSUE THREAD

**Issue by stajics** [vercel/next.js#25647]
### What version of Next.js are you using?

10.2.3

### What version of Node.js are you using?

14.15.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

vercel

### Describe the Bug

Using a new next.js project. Optimized images won't load from a specific domain.  

For 
```
<Image src="https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg" width={500} height={500} />
```
getting 503 (Service Unavailable) error.
<img width="373" alt="Screen Shot 2021-05-31 at 4 46 26 PM" src="https://user-images.githubusercontent.com/17711182/120210642-4c583500-c230-11eb-9f62-fd75c79ac461.png">

Adding `unoptimized` prop shows the image.

### Expected Behavior

Image should load from any domain

### To Reproduce

Create a new next app.
Add 
```
<Image src="https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg" width={500} height={500} />
```
to a page and navigate to the page. Image will be broken

**Comment by manowski** [comment]
I had the same issue and was wondering why it doesn't work. As a result, I discovered that some urls containing images are not in the image format. That's why you're getting an error and the image won't load.
Your image, for example, is in html format and you've put it in the image component.

You can check the format here: aconvert.com/analyze.html

**Comment by stajics** [comment]
@manowski  Makes sense. But I get that the format is `JPG/JPEG` for https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg

**Comment by balazsorban44** [comment]
A simple `curl https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg` shows that you return HTML, instead of an image response.

```js
await fetch(
  "https://valence.jobboard.io/uploads/job/logo/65255312/FreightWaves.jpg"
)
```

Also gives back 503, so the issue is probably not with `next/image`.
